packages/components/src/components/Flex/Flex.tsx

```diff
@@ -15,6 +15,7 @@ import { useElevation } from '../ElevationContext/ElevationContext';
 
 export const allowedFlexFrameProps = [
     'margin',
+    'padding',
```

Comment:
Is it necessary since we have `margin` here?

Reply:
The same as above. Also, I needed it in the AccountGroup refactor, that's a case where margin won't work and we need padding.